### PR TITLE
Spectra v2.7.6

### PR DESCRIPTION
**Pull request by adi3890**
* Fix: Form - Resolved multiple styling issues on the frontend.
* Fix: Image Gallery - Resolved an issue wherein the lightbox feature would not work after setting up pagination.
* Fix: Global - Resolved a significant issue of multiple `_uag_page_assets` entries in the database when using an FSE theme.

### CODE REVIEW

**Review by bsf-zanev — APPROVED** [review]
Once we merge #3425, we're good to go.

**Review by AbhijitNage123 — APPROVED** [review]

File: classes/class-uagb-post-assets.php [hunk 1]
```diff
@@ -355,6 +355,12 @@ public function allow_assets_generation() {
 		// If version is updated, return true.
 		if ( $version_updated ) {
 			// Delete cached meta.
+			$unique_ids = get_option( '_uagb_fse_uniqids' );
+			if ( is_array( $unique_ids ) && ! empty( $unique_ids ) ) {
```
Comment (adi3890) [inline classes/class-uagb-post-assets.php:359]: @AbhijitNage123 empty check should be before is_Array

**Review by rajmohanreddy774 — APPROVED** [review]

File: classes/class-uagb-front-assets.php [hunk 2]
```diff
@@ -100,6 +100,37 @@ public function set_initial_variables() {
...
+			$dynamic_id               = false;
+			$updated_uagb_fse_uniqids = array();
+			// Split the string by the forward slashes.
+			$template_id = explode( '//', $_wp_current_template_id, 2 );
+			// Get the second template name as id after the forward slashes.
+			$template_name             = $template_id[1];
+			$template_id_based_on_name = hash( 'crc32b', $template_name );
+			$dynamic_id                = hexdec( $template_id_based_on_name );
+			$get_uagb_fse_uniqids      = get_option( '_uagb_fse_uniqids' ); // Retrieve the existing array.
+			if ( is_array( $get_uagb_fse_uniqids ) && ! empty( $get_uagb_fse_uniqids ) ) {
```
Comment (adi3890) [inline classes/class-uagb-front-assets.php:122]: empty check should be done before

File: classes/class-uagb-rest-api.php [hunk 3]
```diff
@@ -173,14 +173,19 @@ public function delete_page_assets( $post_id ) {
 				}
 			}
 
+			$unique_ids = get_option( '_uagb_fse_uniqids' );
+			if ( is_array( $unique_ids ) && ! empty( $unique_ids ) ) {
```
Comment (adi3890) [inline classes/class-uagb-rest-api.php:177]: empty check should be first thing

Comment (AbhijitNage123) [reply classes/class-uagb-rest-api.php:177]: Done here https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3426

File: classes/class-uagb-front-assets.php (hunk 2 above)
Comment (AbhijitNage123) [reply classes/class-uagb-front-assets.php:122]: Done here - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3426

File: classes/class-uagb-post-assets.php (hunk 1 above)
Comment (AbhijitNage123) [reply classes/class-uagb-post-assets.php:359]: Done here - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3426

**Review by rahulvarma722 — APPROVED** [review]

**Pull request merged**